README.md

```diff
@@ -57,11 +64,20 @@ console.log('from Base64 String: ', scteMessageFromBase64String(B64_STRING));
 console.log('from Hex String: ', scteMessageFromHexString(HEX_STRING));
 ```
 
-**Android**
 
-**iOS**
 
-After running the build command, the iOS framework can be found here: `build/cocoapods/framework/scte35decoder.framework`. _When adding the framework to the project, be sure to choose "copy if needed."_
+## iOS
+
+
+To create & install a framework to use in iOS/Xcode:
```

Comment:
Awesome steps. Well-written and easy to understand. Feels like even a novice to iOS could follow these.